### DISCUSSION TITLE
Any thought's on supporting @Remix.run framework ?

### DISCUSSION THREAD

**Discussion by AlonGvili** [tinacms/tinacms#2228] [Ideas]
## Summary

[remix.run](https://remix.run/)

It seems it is getting popularity so supporting it doesn't sound like a bad thing.

here is a little comparison 
[remix vs next](https://sergiodxa.com/articles/remix-vs-next-js-comparison)

**Comment by scottgallant** [comment]
Great question, @AlonGvili! This is something that @jeffsee55 and I have discussed in the past. He can speak to this better than I could, but we the short answer is: yes we hope to support Remix in the not-too-distant future. 

Do you use Remix?  Do you think it could be a good match for TinaCMS?

**Comment by jeffsee55** [comment]
Yes I'm a big fan of Remix, the only thing that's preventing us from supporting it right now is that you're currently only able to talk to Tina Cloud while in "edit-mode". This works for Next.js because you can use `getStaticProps`, which runs at build time. So once we support read-only tokens on Tina Cloud we'll be able to support Remix.

**Comment by scottgallant** [comment]
Here's a clip of @jsee showing a Tina + Remix site that he's working on where he's pulling product data from Shopify into Markdown files.  

https://www.youtube.com/watch?v=5irYOzCiZUo

Repo here: https://github.com/jeffsee55/snauwaert-remix
